violet-framework/src/main/java/com/horstmann/violet/workspace/WorkspacePanel.java

```diff
@@ -34,6 +34,8 @@ public void prepareLayout()
         add(scrollSideBarPanel, BorderLayout.WEST);
 //        JScrollPane scrollStatusBarPanel = getScrollableStatusBar();
```

Comment:
why comments inside function? It isn't good practice to use it. To use java/javascript/php/other - doc is better.